src/Altinn.Apps/AppFrontend/react/altinn-app-frontend/src/components/base/AttachmentListComponent.tsx

```diff
@@ -0,0 +1,47 @@
+import * as React from 'react';
+import { AltinnAttachment } from 'altinn-shared/components';
+import { useSelector } from 'react-redux';
+import { IRuntimeState } from 'src/types';
+import { IAttachment } from 'altinn-shared/types';
+import { mapInstanceAttachments } from 'altinn-shared/utils';
+import { Grid, Typography } from '@material-ui/core';
+
+export interface IAttachmentListProps {
+  id: string;
+  text: string;
+  dataTypeIds?: string[];
+}
+
+export function AttachmentListComponent(props: IAttachmentListProps) {
+  const currentTaskId = useSelector(
+    (state: IRuntimeState) => state.instanceData.instance.process.currentTask.elementId,
+  );
+  const dataForTask = useSelector((state: IRuntimeState) => {
+    const dataTypes = state.applicationMetadata.applicationMetadata.dataTypes.filter((type) => {
+      return type.taskId === state.instanceData.instance.process.currentTask.elementId;
+    });
+    return state.instanceData.instance.data.filter((dataElement) => {
+      if (props.dataTypeIds) {
+        return props.dataTypeIds.findIndex((id) => dataElement.dataType === id) > -1;
```

Comment:
Just a note - `findIndex`  is not supported by IE. 
Maybe this is a job for a later time, but we should look into poly filling methods we use to support IE.

Reply:
See that we have documented some of this in #4742 . Added this as a poin to consider.